tracer/src/Datadog.Trace/Debugger/ExceptionAutoInstrumentation/TrackedStackFrameNode.cs

```diff
@@ -127,7 +127,7 @@ private string CreateSnapshot()
                 members = new MethodScopeMembers(0, 0);
             }
 
-            using var snapshotCreator = new DebuggerSnapshotCreator(isFullSnapshot: true, location: ProbeLocation.Method, hasCondition: false, Array.Empty<string>(), members);
+            using var snapshotCreator = new DebuggerSnapshotCreator(isFullSnapshot: true, location: ProbeLocation.Method, hasCondition: false, Array.Empty<string>(), members, maxInfo: default);
```

Comment:
We might should use a real `maxInfo` here because this is going to create a snapshot. Maybe the correct place to initiate `maxInfo` for it is from `ExceptionDebuggingProcessor.UpdateProbeProcessor`

Reply:
Ahhh, I see. 
However, since we create this probe by our self, and not from backend we should use default values.
I fixed it here: 8982541aa0fe89abb99bd1a73d0a20267e1e688d